src/fondant/compiler.py

```diff
@@ -147,16 +147,12 @@ def _generate_spec(
 
             component_op = component["fondant_component_op"]
 
-            # add metadata argument to command
-            command = ["--metadata", json.dumps(asdict(metadata))]
-
-            # add in and out manifest paths to command
-            command.extend(
-                [
-                    "--output_manifest_path",
-                    f"{path}/{safe_component_name}/manifest.json",
-                ],
-            )
+            command = [
+                "--metadata",
+                json.dumps(asdict(metadata)),
+                "--output_manifest_path",
+                f"{path}/{safe_component_name}/manifest.json",
+            ]
```

Comment:
This will conflict with some of the changes in your other PRs.

Reply:
True, will revert it back